#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_return-deffn.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## The "return" keyword will appear in the function index, instead of the
## variable index

@DPATCH@

--- octave2.1-2.1.73.orig/doc/interpreter/func.txi
+++ octave2.1-2.1.73/doc/interpreter/func.txi
@@ -373,12 +373,12 @@
 without adding extra logic to avoid printing the message if the vector
 does contain a nonzero element.
 
-@defvr {Keyword} return
+@deffn {Keyword} return
 When Octave encounters the keyword @code{return} inside a function or
 script, it returns control to the caller immediately. At the top level,
 the return statement is ignored.  A @code{return} statement is assumed
 at the end of every function definition.
-@end defvr
+@end deffn
 
 @DOCSTRING(return_last_computed_value)
 
